warehouse/accounts/views.py

```diff
@@ -657,6 +657,10 @@ def _error(message):
 
     # Check whether the user has logged in since the token was created
     last_login = data.get("user.last_login")
+    # Before updating itsdangerous to 2.x the last_login was naive,
+    # now it's localized to UTC
+    if last_login[-6] != "+":
+        last_login += "+00:00"
```

Comment:
This feels a bit fragile. Instead of comparing strings here (and possibly, below) could we compare datetimes and check for tzinfo instead?

Reply:
Sure, I considered the datetimes for a bit, but wasn't sure whether there was something more behind string comparison originally 🤔